osm/src/test/groovy/org/orbisgis/orbisprocess/geoclimate/osm/FormattingForAbstractModelTests.groovy

```diff
@@ -12,7 +12,7 @@ import static org.junit.jupiter.api.Assertions.assertTrue
 
```

Comment:
It seems that you have edited a file which should not be changed. Maybe I'm wrong.